.github/workflows/v_apps_and_modules_compile_ci.yml

```diff
@@ -119,6 +119,17 @@ jobs:
           echo "Execute Tests"
           cd /tmp/adventofcode && v run verify.v
 
+      - name: Build vlang/msgpack
+        run: |
+          echo "Install msgpack"
+          v install msgpack
+          echo "Build msgpack"
+          v ~/.vmodules/msgpack/
+          echo "Run msgpack tests"
+          VJOBS=1 v -stats test ~/.vmodules/msgpack/
```

Comment:
Why `VJOBS=1` ? It works imho fine without it, since the test files there are independent from each other.